babel.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+	plugins: [
+		'@babel/plugin-transform-runtime',
+		'@babel/plugin-transform-react-jsx'
```

Comment:
@juliushaertl do we really need this? Added in https://github.com/nextcloud/nextcloud-vue/pull/439